examples/provider/provider.tf

```diff
@@ -20,57 +20,96 @@ data "anxcloud_core_location" "anx04" {
   code = "ANX04"
 }
 
-data "anxcloud_template" "anx04" {
-  location_id   = data.anxcloud_core_location.anx04.id
-  template_type = "templates"
-}
-
-locals {
-  debian_11_templates = values({
-    for i, template in data.anxcloud_template.anx04.templates : tostring(i) => template
-    if substr(template.name, 0, 9) == "Debian 11"
-  })
-}
-
 resource "anxcloud_vlan" "example" {
-  description_customer = "example-terraform"
-  location_id          = data.anxcloud_core_location.anx04.id
-  vm_provisioning      = true
+  location_id     = data.anxcloud_core_location.anx04.id
+  vm_provisioning = true
 }
 
-resource "anxcloud_network_prefix" "example" {
+resource "anxcloud_network_prefix" "v4" {
   vlan_id     = anxcloud_vlan.example.id
   location_id = data.anxcloud_core_location.anx04.id
   ip_version  = 4
-  type        = 0
-  netmask     = 29
+  netmask     = 30
 }
 
-resource "anxcloud_virtual_server" "example" {
-  hostname      = "example-terraform"
-  location_id   = data.anxcloud_core_location.anx04.id
-  template_id   = local.debian_11_templates[0].id
-  template_type = "templates"
+resource "anxcloud_network_prefix" "v6" {
+  vlan_id     = anxcloud_vlan.example.id
+  location_id = data.anxcloud_core_location.anx04.id
+  ip_version  = 6
+  netmask     = 126
+}
+
+resource "anxcloud_ip_address" "v4" {
+  address           = cidrhost(anxcloud_network_prefix.v4.cidr, 2)
+  network_prefix_id = anxcloud_network_prefix.v4.id
+}
+
+resource "anxcloud_ip_address" "v6" {
+  address           = cidrhost(anxcloud_network_prefix.v6.cidr, 2)
+  network_prefix_id = anxcloud_network_prefix.v6.id
+}
+
+resource "anxcloud_virtual_server" "webserver" {
+  hostname    = "terraform-playground"
+  location_id = data.anxcloud_core_location.anx04.id
+  template    = "Debian 11"
 
   cpus   = 4
   memory = 4096
 
   ssh_key = file("~/.ssh/id_rsa.pub")
 
+  script = <<-EOT
+    #!/bin/bash
+
+    # install nginx server
+    apt update && apt install -y nginx
+    EOT
+
+  network {
+    vlan_id  = anxcloud_vlan.example.id
+    ips      = [anxcloud_ip_address.v4.id]
+    nic_type = "vmxnet3"
+  }
+
   network {
     vlan_id  = anxcloud_vlan.example.id
+    ips      = [anxcloud_ip_address.v6.id]
```

Comment:
are we now adding two interfaces, one for IPv4 and one for IPv6? `ips` looks like an array, shouldn't we add both v4 and v6 to a single array (and interface I guess?)?